src/dash/models/DashManifestModel.js

```diff
@@ -292,6 +292,17 @@ function DashManifestModel(config) {
         return adaptation[DashConstants.CENC_DEFAULT_KID];
     }
 
+    function getLabel(adaptation) {
+        let label;
+
+        if (adaptation && adaptation.hasOwnProperty(DashConstants.LABEL) && adaptation.label) {
```

Comment:
I do believe this will only grab the label attribute of the `<AdaptationSet>`. Unless I misunderstood, what's been posted from the IOP specifies that it's actually `<Label>` elements that are children of `<AdaptationSet>`. So this should change to finding all `<Label>` children, and then picking one (perhaps by choosing the `<Label>` with its `lang` attribute set to the browser locale).

Reply:
@squarebracket  Updated.